.github/dependabot.yaml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  # Upgrade dependencies of GitHub Actions.
```

Comment:
Nit: I still think this is fairly self-evident since the `package-ecosystem` is named `github-actions`.